packages/integration-react/src/api/ScmAuth.ts

```diff
@@ -199,13 +199,16 @@ export class ScmAuth implements ScmAuthApi {
     const host = options?.host ?? 'dev.azure.com';
     return new ScmAuth('azure', microsoftAuthApi, host, {
       default: [
-        'vso.build',
-        'vso.code',
-        'vso.graph',
-        'vso.project',
-        'vso.profile',
+        '499b84ac-1321-427f-aa17-267ca6975798/vso.build',
+        '499b84ac-1321-427f-aa17-267ca6975798/vso.code',
+        '499b84ac-1321-427f-aa17-267ca6975798/vso.graph',
+        '499b84ac-1321-427f-aa17-267ca6975798/vso.project',
+        '499b84ac-1321-427f-aa17-267ca6975798/vso.profile',
+        'profile',
+        'openid',
+        'email',
```

Comment:
I'd like to know more about why these are needed? Are these not possible to add using `additionalScopes`? Maybe @afscrome can clarify as well?

Reply:
The `Microsoft profile contained no email` issue should have been fixed in #23032 . If you're still getting that issue, we should take a look at that again and fix it properly, rather than work around it for just this provider.

Reply:
Probably would be good to rebase on latest then based on @afscrome's comment and remove those scopes to see what the results are

Reply:
You're absolutely right guys - I must've missed the latest changes. Thanks for catching that one!

I've pushed my latest changes omitting those scopes. I've tested it and looks good from here!

Reply:
Awesome! Pretty sure that was fixed within hours of the `1.23.0` release going out on Tuesday

Reply:
On the failing test run, it might be worth re-triggering. I'm going to use those famous last words...  I can't replicate it on my machine 🤭 .